pkg/cli/admin/release/info.go

```diff
@@ -328,23 +337,29 @@ func replaceClusterSemanticArgs(f kcmdutil.Factory, args []string, semanticArgs
 	return args, nil
 }
 
-func findArgumentsFromCluster(f kcmdutil.Factory, args []string) ([]string, error) {
+func findArgumentsFromCluster(f kcmdutil.Factory, args []string, grapharch string) ([]string, error) {
 	semanticArgs := findSemanticVersionArgs(args)
 	if len(semanticArgs) == 0 && len(args) > 0 {
 		return args, nil
 	}
 	klog.V(4).Infof("Found semantic versions: %v", semanticArgs)
-	// attempt to find semantic args from the cluster
-	args, clusterErr := replaceClusterSemanticArgs(f, args, semanticArgs)
-	if len(semanticArgs) == 0 {
-		return args, clusterErr
-	}
-	// if any semantic args remain, try to fetch them from the api endpoint out of a stable channel
-	err := replaceStableSemanticArgs(args, semanticArgs, defaultGraphURL)
-	if len(semanticArgs) == 0 || err != nil {
+
+	//if graph-arch isn't defined, attempt to find semantic args from the cluster
+	if grapharch == "" {
```

Comment:
shouldn't we also try the cluster and check if the arch is equal to grapharch?

Reply:
the reasoning was that if a user specifies the arch, it makes sense to directly retrieve from the graph?

Reply:
that is true, but in a sense, --arch doesn't make sense when querying a cluster because it will be only a single arch. so either it matches or it doesn't. moreover, there is no easy way to detect the architecture of a cluster...the only way is to look at node labels to figure out their architecture.

Reply:
ok, I suppose if we describe it clearly in help

Reply:
How does this look?: 

"Specify a release's architecture (amd64|arm64|ppc64le|s390x). When used, it will bypass searching the current connected cluster for the release payload."

Reply:
+1 sounds good